pkg/ocm/helpers.go

```diff
@@ -354,9 +356,19 @@ func (c *Client) LinkOrgToRole(orgID string, roleARN string) (bool, error) {
 	return true, nil
 }
 
+func (c *Client) IsRoleLinked(orgID string, roleARN string) (bool, error) {
+	resp, err := c.ocm.AccountsMgmt().V1().Organizations().Organization(orgID).
+		Labels().Labels(STSOCMRole).Get().Send()
+	if err != nil {
```

Comment:
@zgalor @pvasant I wasn't sure if in the case of **404** or **403**: 
Should we ignore the error and return `linked==false` for this particular role? 
Or return the error and exit, so no ocm-roles are returned to the user? in case we can't figure out if a certain role is linked.

Reply:
404 means no label - so linked=false.
You shouldn't be getting 403 when getting the user org. Did that happen or are you just asking hypothetically?

Reply:
That right! thanks.
Didn't happen, just asking hypothetically.